### ISSUE TITLE
Invalid ISO time from wikidataTimeToISOString

### ISSUE THREAD

**Issue by moshest** [maxlath/wikidata-sdk#26]
Please consider the following date:

```js
> wdk.wikidataTimeToISOString('-34000-00-00T00:00:00Z')
'+033999-12-31T22:00:00.000Z'
```
The return value is in the future instead of the past.

This Wikidata time return from the item "dog":
https://www.wikidata.org/wiki/Special:EntityData/Q144.json

**Comment by maxlath** [comment]
vanilla JS and big dates are a bad matches, maybe we need this kind of work around https://stackoverflow.com/a/34897918/3324977

**Comment by moshest** [comment]
I disagree.

The output for this value should be: `"-034000-01-01T00:00:00Z"`

```js
> new Date('-034000-01-01T00:00:00Z')
-034000-01-01T00:00:00.000Z
```

**Comment by maxlath** [comment]
just published `v5.9.5` with a patch (f5a2f9a) that should fix it: the issue is that [years with more than 4 digits need different 0-padding](https://github.com/maxlath/wikidata-sdk/commit/f5a2f9a7257d4c3e4ef50175fdefdd121d6909d7#diff-3cfdf338c62764b22bc6993220999b60). Can you confirm that it works for you?

**Comment by moshest** [comment]
Fixed. Thanks!
